ldk/javascript/.eslintrc.js

```diff
@@ -20,6 +20,8 @@ module.exports = {
     // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-shadow.md
     'no-shadow': 'off',
     '@typescript-eslint/no-shadow': ['error'],
+    "no-use-before-define": "off",
+    "@typescript-eslint/no-use-before-define": ["error"]
```

Comment:
I'd much rather suppress the rule in a given instance unless we have a good reason to suppress it overall.

Reply:
Nope this was my error b/c I didn't read the change closely!